src/main/java/com/parkinglotsystem/ParkingFloor.java

```diff
@@ -0,0 +1,13 @@
+package com.parkinglotsystem;
+
+import java.util.ArrayList;
+
+public class ParkingFloor {
```

Comment:
Don't keep all files in root dir. Create new directories for each type of classes if required. Ex:
- Models
- Services
- Controllers/Handlers
- CustomExceptions
etc.